homeassistant/components/geocaching/sensor.py

```diff
@@ -0,0 +1,74 @@
+"""Platform for sensor integration."""
+from __future__ import annotations
+
+from homeassistant.components.geocaching.coordinator import (
+    GeocachingDataUpdateCoordinator,
+)
+from homeassistant.components.sensor import SensorEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_ICON
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import (
+    ATTR_DEFAULT_ENABLED,
+    ATTR_DEVICE_CLASS,
+    ATTR_NAME,
+    ATTR_SECTION,
+    ATTR_STATE,
+    ATTR_UNIT_OF_MEASUREMENT,
+    DOMAIN,
+    SENSOR_ENTITIES,
+)
+from .models import GeocachingEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistantType, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up a Geocaching sensor entry."""
+
+    coordinator = hass.data[DOMAIN][entry.entry_id]["COORD"]
+    async_add_entities(
+        [GeocachingSensor(coordinator, key=key) for key in SENSOR_ENTITIES], True
```

Comment:
No need to pass `True` since coordinator has data from setup_entry.
```suggestion
        [GeocachingSensor(coordinator, key=key) for key in SENSOR_ENTITIES]
```